clients/common/utility.cpp

```diff
@@ -60,6 +78,28 @@ int type2int<hipblasDoubleComplex>(hipblasDoubleComplex val)
 // Return path of this executable
 std::string hipblas_exepath()
 {
+#ifdef WIN32
+    std::vector<wchar_t> result(MAX_PATH + 1);
+    // Ensure result is large enough to accomodate the path
+    for(;;)
+    {
+        auto length = GetModuleFileNameW(nullptr, result.data(), result.size());
+        if(length < result.size() - 1)
+        {
+            result.resize(length);
+            result.shrink_to_fit();
+            break;
+        }
+        result.resize(result.size() * 2);
+    }
+
+    std::wstring wspath(result.data());
+    std::filesystem::path exepath(wspath.begin(), wspath.end());
```

Comment:
This section should be changed to use the non UNICODE GetModuleFileNameA call for now, unless you decide to define UNICODE.   See other more recent changes in rocblas.  Can discuss but probably can get windows CI running as as long as you don't break linux this can be determined later.